### PR TITLE
132/llama3.2+nomic

### PR DESCRIPTION
**Pull request by jfrverdasca**
## What this PR change:

* Add ollama local embeddings support for fast testing
* Fix docker compose and docker file issues
* Fix tests to accommodate changes when refactoring embeddings
* Fix makefile instructions and help
* Add makefile instruction do pull ollama models

## Reviewers:

- [ ] @kenvontucky 
- [ ] @JdFSilva

## Issues:

* #132 
* #137 

## Print screens showing the changes:

NA

### CODE REVIEW

**Review by kenvontucky — CHANGES_REQUESTED** [review]
Please add a comment in Readme showcasing how we can set this up for llama3.2 and nomic

File: labs/embeddings/vectorizers/chunk_vectorizer.py [hunk 1]
```diff
@@ -67,20 +63,20 @@ def load_docs(self, root_dir, file_extensions=None):
...
 
         logger.debug("Embedding all repo documents.")
-        embeddings = embedding(model="text-embedding-ada-002", input=texts)
+
+        embedder = Embedder(OpenAIEmbedder)
+        embeddings = embedder.embed(prompt=texts)
 
         logger.debug("Storing all embeddings.")
-        reembed_code(files_and_texts, embeddings, repo_destination)  # type: ignore
+        embedder.reembed_code(files_texts=files_and_texts, embeddins=embeddings, repository=repo_destination)  # type: ignore
```
Comment (JdFSilva) [inline labs/embeddings/vectorizers/chunk_vectorizer.py:82]: ```embeddins=embeddings```
is missing the G
```embeddings=embeddings```

File: labs/tasks/llm.py [hunk 2]
```diff
@@ -55,6 +58,7 @@ def get_llm_response_task(prefix="", context={}):
     llm_response = get_llm_response(context)
 
     if prefix:
-        redis_client.set(f"{prefix}_llm_response", llm_response[1][1].choices[0].message.content)
+        logger.debug(f"{llm_response=}")
```
Comment (JdFSilva) [inline labs/tasks/llm.py:61]: is this supposed to be here or was it used just for local testing?

**Review by JdFSilva — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -37,5 +37,7 @@ Here's the steps to setup the project locally:
 1. `poetry shell`
 2. `poetry install`
 3. `make up`
-4. `make api` or `ENV=local make api`
-5. `ENV=test make tests`
+4. `make ollama model=nomic-embed-text:latest`
```
Comment (kenvontucky) [inline README.md:40]: lets make it a specific entry like

### Llama 3.2 with nomic

1. `make ollama model=nomic-embed-text:latest`
2. make ollama model=llama3.2:latest`
3. `make api` or `ENV=local make api`

**Review by kenvontucky — APPROVED** [review]

**Pull request merged**